packages/react-components/react-components/stories/Concepts/Positioning/PositioningDisableTransform.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Button, Popover, PopoverSurface, PopoverTrigger } from '@fluentui/react-components';
+
+export const DisableTransform = () => {
+  return (
+    <Popover positioning={{ useTransform: false }}>
+      <PopoverTrigger disableButtonEnhancement>
+        <Button appearance="primary">Click me</Button>
+      </PopoverTrigger>
+
+      <PopoverSurface style={{ minWidth: 100 }}>Container</PopoverSurface>
+    </Popover>
+  );
+};
+
+DisableTransform.parameters = {
+  docs: {
+    description: {
+      story: [
+        'By default, the positioned element is positioned using transform in the element style for better performance, but can be disabled by setting `useTransform` to `false`.',
+        '',
+        'Note:',
+        'When disabling transforms, add `width: max-content` (or a fixed width) to the floating element to prevent positioning issues due to layout affecting the floating element ([Disabling transform](https://floating-ui.com/docs/react#disabling-transform)).',
```

Comment:
Hi @layershifter , I am not quite sure if this 
`Note: (When disabling transforms, add width: max-content (or a fixed width) to the floating element to prevent positioning issues due to layout affecting the floating element)` is still needed in our case? 

I tried without setting width in storybook, I don't see any difference, maybe my example is quite simple. Am I missing any potential cases for this note? Or fluentui already improves this? 

Thank you for sharing more insights about this.

Reply:
I'm not entirely sure in what case the `with: max-content` would be necessary and Floating UI doesn't seem to specify any examples. I would say to remove this guidance until we have actual issues reported by users.

If you can think of an example where this would be a problem, then I'd be happy to include this guidance in the docs with that example

Reply:
Thank you for suggestion. Yeah I cannot think of one example for now, removed it and will keep it in mind if there's any issue about it.